libkineto/src/CuptiActivityProfiler.cpp

```diff
@@ -840,6 +840,14 @@ void CuptiActivityProfiler::configure(
       timestamp = system_clock::now();
     }
 #ifdef HAS_CUPTI
+#ifdef _WIN32
+    CUPTI_CALL(
+        cuptiActivityRegisterTimestampCallback([]() -> uint64_t {
```

Comment:
I'm wondering if roctracer also needs a similar timestamp callback change. @mwootton , @jeffdaily

Reply:
I couldn't find the hip equivalent to this CUPTI API: https://docs.nvidia.com/cupti/modules.html#group__CUPTI__ACTIVITY__API_1gc168c4a9b5f4e9bd7553ca26118a307d

Reply:
Unfortunately, I do not have an amd gpu to check for roctracer. Anything else I could do to help move this forward?

Reply:
We have some internal CI , let me try importing in there.